07-routing-gossip.md

```diff
@@ -422,10 +433,6 @@ Notice that this is a node policy and MUST NOT be enforced by peers, e.g., by cl
 
 ## Recommendations for Routing
 
-As the fee is proportional, it must be calculated backwards from the
-destination to the source: only the amount required at the final
```

Comment:
Why was this removed?

Reply:
Ah ok I see it was moved above!

Reply:
Yes I thought it made more sense there and it wasn't really a recommendation anyway.